### ISSUE TITLE
Documentation: How to set up a profile

### ISSUE THREAD

**Issue by AndrewVaughn** [rsanchez-wsu/RaiderPlanner#304]
As I used this planner to set up a profile, I found 0 documentation to assist with this task. The only help I found was on the Website. Its only advise was download the program, click on RaiderPlanner.git and go.
There should be full documentation on how to properly and easily use this program. 
This could be accomplished by linking instructions to the website or the help header on the navigation menu.

**Comment by rsanchez-wsu** [comment]
Yes, please!  We desperately need to improve the application documentation.

**Comment by AndrewVaughn** [comment]
I will work on this
